README.md

```diff
@@ -56,6 +56,18 @@ To get started, please visit our [documentation](https://docs.paradedb.com).
 
 ParadeDB and its extensions are available as commercial software for installation on self-hosted Postgres deployment and via Docker and Kubernetes as standalone images. For more information, including enterprise features and support, please [contact us by email](mailto:sales@paradedb.com).
 
+### Using the Setup Script
+
+For an interactive setup run the following in your terminal.
```

Comment:
I did some experimenting, and using the ClickHouse script it does this:

```bash
philippemnoel /Users/philippemnoel/Downloads/paradedb/paradedb (phil/docs) $ curl https://clickhouse.com/ | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2822    0  2822    0     0  14432      0 --:--:-- --:--:-- --:--:-- 14471

Will download https://builds.clickhouse.com/master/macos-aarch64/clickhouse into clickhouse

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 70.4M  100 70.4M    0     0  1287k      0  0:00:56  0:00:56 --:--:-- 1198k

Successfully downloaded the ClickHouse binary, you can run it as:
    ./clickhouse
```

It doesn't seem to be possible to preserve the interactive nature. A simpler command to yours here can simply be:

```bash
curl -O https://raw.githubusercontent.com/destrex271/paradedb/feature/setup_script/setup.sh && bash setup.sh
```

That works for me. This way we can keep it as a single line?

Reply:
Yep this works too! I'll update the readme.
Anything else that we need to review after this?

Reply:
All done. 
Please do tell me if any other changes are required!

Reply:
Thank you!